cypress/support/helpers.js

```diff
@@ -259,3 +259,63 @@ export const responsiveAssertHelper = ( callback = () => {}, options = {} ) => {
 
 	return responsiveAssertFunctions
 }
+
+/*
+* Helper function for Typography popover assertion.
+*
+* @param {string} selector
+* @param {string} viewport
+*/
+export const assertTypography = ( selector, viewport ) => {
```

Comment:
future proof the function. pass `options` object instead of `viewport` as argument.